soroban-sdk/src/env.rs

```diff
@@ -469,6 +469,79 @@ use std::{path::Path, rc::Rc};
 #[cfg(any(test, feature = "testutils"))]
 use xdr::{LedgerEntry, LedgerKey, LedgerKeyContractData, SorobanAuthorizationEntry};
 
+#[cfg(any(test, feature = "testutils"))]
+pub struct TestStellarAssetContractUtil {
```

Comment:
Can you move this type and it's implementation into the `testutils` package? You can drop the `Test` prefix also, since none of the other types in the testutils package have the name redundancy.

Reply:
Done